### DISCUSSION TITLE
couchDB connection problems

### DISCUSSION THREAD

**Discussion by RASS00** [Budibase/budibase#4833] [Help] [answered]
Hi, I can't seem to be able to connect to couchdb, as it refuses my connection everytime, it refuses to return the GET, ports 4005, 5984, 4369 & 9100 are unlocked via ufw, I'm using ubuntu 21.10 x64, with amd epyc 2 chips, I used the CLI with docker & docker-compose to setup bb
![image](https://user-images.githubusercontent.com/82243352/157072393-71c8f73e-ebad-4b05-bff8-21ffc5734af6.png)

I have fauxton installed and i host in DO but I DID NOT use the one click deployment droplet

![image](https://user-images.githubusercontent.com/82243352/157073250-57850550-e9d0-4e3f-8985-90199ec26dbe.png)
Budi hosting status

I have not edited any of the build settings as of now, I have both .env and docker-compose.yaml files present

My bash_history is as follows:
apt upgrade
apt upgrade -a
apt install upgrade
full-upgrade
apt full-upgrade
apt install npm
apt update
apt full-upgrade
apt install npm
apt install nodejs
npm i -g @budibase/cli
ls -a
budi
budi hosting --init
budi hosting --start
apt install docker docker-compose
budi hosting --start
budi hosting --init
budi hosting --start
ufw allow 5984
ufw allow 4005
ufw allow 10000
ufw allow 9100
sudo shutdown -r now

**Comment by shogunpurple** [comment, marked as answer]
Hi @RASS00 - as you can see from your ports, the only port that is exposed on the host machine (your DO droplet) is 10000. There is a rule in our `nginx` config which should give you fauxton access.

`<your-droplet-ip>:10000/db/_utils`

Can you see fauxton if you visit that URL?

**Reply by RASS00** [reply]
@shogunpurple it returns this
![image](https://user-images.githubusercontent.com/82243352/157086273-ba0cc93a-5917-4acd-b8a7-e15b2e093119.png)
have also tried IP:10000/_utils & /db 
I also tried to disable ufw to acess and still nothing
I have also tried a variations of different ports (4005 & 5984, mostly to no sucess) have this problem since yesterday, "first" time setup

budibase (ip:10000) is running correctly

**Reply by shogunpurple** [reply]
why is it redirecting to HTTPS? Did you set up HTTPS on your server?

**Reply by RASS00** [reply]
@shogunpurple omg, so sorry I think it was because firefox was blocking it (I have strict privacy settings), changed to chrome and it worked.
